python/pyarrow/table.pxi

```diff
@@ -51,9 +79,27 @@ cdef class ChunkedArray(_PandasConvertible):
 
     @property
     def type(self):
+        """
+        Examples
```

Comment:
Small comment for future reference: best to add a first line (which is not the "Examples" title), eg something like "The data type of the chunked array" in this case, because this first line is being used by autosummary to include in the summary table (eg the tables at https://arrow.apache.org/docs/python/generated/pyarrow.ChunkedArray.html)

Reply:
Oh, true! Will make a MINOR PR to correct this. Thanks!

Reply:
[MINOR: [Python] Add description to pa.ChunkedArray.type docstring](https://github.com/apache/arrow/pull/12832)